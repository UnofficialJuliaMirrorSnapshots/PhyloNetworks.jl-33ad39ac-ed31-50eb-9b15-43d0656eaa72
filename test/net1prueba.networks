((2,(#H-5:::0.49,1)):1.5121335305466412,4,((5,(6)#H-5:::0.51):0.5191605811059357,3):0.489418799149601);, with -loglik -1.0
((2,((#H-3:::0.49,6),1)):1.5121335305466412,4,((5)#H-3:::0.51,3):0.489418799149601);, with -loglik -1.0
((2,(((5,#H-4:::0.49):10.0,6),1)):1.5121335305466412,4,(3)#H-4:::0.51);, with -loglik -1.0
((5,(#H-2:::0.49,3):0.5191605811059357):10.0,6,((2,(4)#H-2:::0.51),1));, with -loglik -1.0
((2)#H8:::0.51,4,((5,(6,(#H8:::0.49,1)):10.0):0.5191605811059357,3):0.489418799149601);, with -loglik -1.0
Problem found when optimizing branch lengths for some networks, left loglik as -1. Please report this issue to claudia@stat.wisc.edu, google group or github issues. Thank you!