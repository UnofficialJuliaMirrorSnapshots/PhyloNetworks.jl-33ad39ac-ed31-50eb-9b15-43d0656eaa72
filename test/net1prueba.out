((2,(1)#H7:::0.7143969563530107):1.5121335305466412,4,((5,(6,#H7:::0.2856030436469893):10.0):0.5191605811059357,3):0.489418799149601); -Ploglik = 126.74250981975307
 Dendroscope: ((2,(1)#H7):1.5121335305466412,4,((5,(6,#H7):10.0):0.5191605811059357,3):0.489418799149601);
 Elapsed time: 8.047296065 seconds in 5 successful runs
-------
List of estimated networks for all runs:
 ((2,(1)#H7:::0.7143969563530107):1.5121335305466412,4,((5,(6,#H7:::0.2856030436469893):10.0):0.5191605811059357,3):0.489418799149601);, with -loglik 126.74250981975307
 (1,((4,#H7:7.599603390006409::0.37302486593984974):0.40021990523139966,(3,(6,5):0.33534692016222695):0.12573257042944036):0.8082647558877085,(2)#H7:10.0::0.6269751340601503);, with -loglik 155.49625734566945
 ((3,(2,#H7:0.0::0.49578624417628053):9.864343470186494):0.13851456440978557,(6,5):0.24980503322268607,((1,4):0.0)#H7:0.0::0.5042137558237194);, with -loglik 192.1045202146181
 ((((5,6):0.3353472312700546,3):0.12573315221965528,(1,(2)#H9:1.3329881588275758::0.6269752696229017):0.808264724878358):0.40021960769620457,4,#H9:4.36389880263218::0.37302473037709827);, with -loglik 155.49625734551378
 ((1,(2)#H7:8.721421786168476::0.6269753032429941):0.8082647064511037,(3,(5,6):0.3353472221500118):0.1257332140052616,(4,#H7:0.0::0.3730246967570059):0.4002195296586772);, with -loglik 155.49625734551572
-------
